stdlib/pervasives.mli

```diff
@@ -1091,6 +1091,15 @@ val at_exit : (unit -> unit) -> unit
    The functions are called in 'last in, first out' order:
    the function most recently added with [at_exit] is called first. *)
 
+(** {6 Misc} *)
+
+val enable_runtime_warnings: bool -> unit
```

Comment:
> set_runtime_warnings doesn't sound right, it's not the warnings that one sets, but their enabled state.

This should read as you assign the `runtime_warnings` variable, i.e. the one accessed through the `runtime_warnings` function. But I don't really mind.